packages/js/components/src/select-control/menu.scss

```diff
@@ -0,0 +1,17 @@
+.woocommerce-select-control__menu {
+    position: absolute;
+    width: 100%;
+    top: 100%;
+    box-sizing: border-box;
+    display: none;
```

Comment:
We should add a `z-index: 10` here as a checkbox shows on top of the list.
Also add a margin-top: $gap-smaller; (is 8px) as the gap seem larger at the moment.

Reply:
Is this in relation to the categories?  I think we should save that for a follow-up PR.  There's still some styling that needs to be tweaked, but I'd like to see how we plan to use for consumption and make some decisions around the component exports before doing it in the base PR (here).

Reply:
Yeah when I don't have the `z-index` the checkbox shows up above the dropdown items. But I think in general a z-index makes sense (the last select-control also had this).